formula-android/src/main/java/com/instacart/formula/integration/FeatureEvent.kt

```diff
@@ -0,0 +1,12 @@
+package com.instacart.formula.integration
+
+import com.instacart.formula.android.Feature
+import com.instacart.formula.fragment.FragmentKey
+
+sealed class FeatureEvent() {
```

Comment:
Adding a sealed class to handle various `Feature` states such as initialization failure or missing binding.